### DISCUSSION TITLE
Can we (or will we) be able to dockerize .NET MAUI apps, so our UI can execute from a container.

### DISCUSSION THREAD

**Discussion by chrisnurse** [dotnet/maui#1053] [Ideas]
I saw that the latest WSL improvements enabled linux UI apps to execute on Windows machines, and have full UI availability.

I want to be a able to containerize an app, built on my Windows machine, where my team mates can pull the container to windows or mac, and run it with no fuss.

Is this a concept on the roadmap?

**Comment by ske66** [comment]
I could be very wrong but I would imagine that .net 6 would be the target for your container. Because .net5 web apps can be dockerised, I would imagine .net6 unified applications would also be dockerisable. But I could be well off

**Reply by chrisnurse** [reply]
Yep .NET 6 naturally. So fingers and toes crossed for this. It would provide a super clean deployment method for apps. Kind of like the old AppV?
